README.md

```diff
@@ -151,6 +168,24 @@ set_namespace("MyApplication")
 
 Flushes the current MetricsContext to the configured sink and resets all properties, dimensions and metric values. The namespace and default dimensions will be preserved across flushes.
 
+The default behavior is to clear all custom dimensions (dimensions added by `put_dimension`) across each flush(), but this can be configured by invoking `logger.flush_preserve_dimensions = True`.
```

Comment:
This might be a bit confusing since users will think that dimensions will only be cleared if using `put_dimensions`.

Reply:
Resolved.